src/main.zig

```diff
@@ -142,6 +145,53 @@ fn help(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.All
     try std.io.getStdOut().writer().print("\n", .{});
 }
 
+fn init() Command_Error!void {
+    // not experienced in zig enough to try this but i feel like you could initalize
+    // both at the same time and in the switch(i) check if both files exist and only if
+    // both conditions are false, continue with creating the rest of the files
+    var main_orng = try std.fs.cwd().createFile("main.orng", .{ .exclusive = true}) catch |i| {
+    switch (i) {
+        i.PathAlreadyExists => {
+            std.log.err("Main.orng already present.", .{});
+            return 1;
```

Comment:
In zig, you indicate an error by returning a value from the error enum, rather than returning `1`. This should be `return error.BuildOrngError`.